packages/playwright/tsup.config.ts

```diff
@@ -1,23 +1,45 @@
 import { defineConfig } from 'tsup';
 
-export default defineConfig((options) => ({
-  entry: [
-    'src/index.ts',
-    'src/bin/archive-storybook.ts',
-    'src/bin/build-archive-storybook.ts',
-    'src/storybook-config/*.ts',
-  ],
-  splitting: false,
+const common = (options) => ({
   minify: !options.watch,
-  format: ['cjs', 'esm'],
+  splitting: false,
   dts: {
     resolve: true,
   },
   treeshake: true,
   sourcemap: true,
   clean: true,
-  platform: 'node',
   esbuildOptions(options) {
     options.conditions = ['module'];
   },
-}));
+});
+
+export default defineConfig((options) => [
+  // We want the playwright functions to be importable from both CJS or ESM files
```

Comment:
I wonder if this addresses https://linear.app/chromaui/issue/AP-4023/fix-esm-build

Reply:
I think it's a great chance that this does.

Reply:
Let's hope so!